src/helix-ui/elements/HXRadioSetElement.js

```diff
@@ -0,0 +1,46 @@
+import { HXElement } from './HXElement';
+import { STATE } from './HXFormControlElement';
+
+/**
+  * Defines behavior for the `<hx-radio-set>` element.
+  *
+  * @extends HXFormControlElement
+  * @hideconstructor
+  * @since 0.16.0
+  */
+export class HXRadioSetElement extends HXElement {
+    static get is () {
+        return 'hx-radio-set';
+    }
+
+    $onConnect () {
+        this.addEventListener('hxchange', this._onHxChanged);
```

Comment:
Please update callback method name to reflect the event. 

```javascript
this.addEventListener('hxchange', this._onHxchange);
```

Reply:
callback method name updated